src/common/client/ip/client.go

```diff
@@ -73,3 +75,11 @@ func (m MainClient) IpBlocksGetById(ipBlockId string) (*ipapisdk.IpBlock, *http.
 func (m MainClient) IpBlocksIpBlockIdDelete(ipBlockId string) (*ipapisdk.DeleteIpBlockResult, *http.Response, error) {
 	return m.IpBlocksApiClient.IpBlocksIpBlockIdDelete(context.Background(), ipBlockId).Execute()
 }
+
+func (m MainClient) IpBlocksIpBlockIdPatch(ipBlockId string, ipBlockPatch ipapisdk.IpBlockPatch) (*ipapisdk.IpBlock, *http.Response, error) {
+	return m.IpBlocksApiClient.IpBlocksIpBlockIdPatch(context.Background(), ipBlockId).IpBlockPatch(ipBlockPatch).Execute()
+}
+
+func (m MainClient) IpBlocksIpBlockIdTagsPut(ipBlockId string) (*ipapisdk.IpBlock, *http.Response, error) {
+	return m.IpBlocksApiClient.IpBlocksIpBlockIdTagsPut(context.Background(), ipBlockId).Execute()
```

Comment:
I think the put request body was missed here... or does the SDK not offer a method for that?